sympy/vector/vector.py

```diff
@@ -40,6 +40,34 @@ def components(self):
         # subclass of Vector the instance belongs to.
         return self._components
 
+    @property
+    def projections(self):
```

Comment:
What about just returning a 3-tuple? Or maybe a coordinate system and a 3-tuple? I'm still a bit skeptical about the return value.

I would also make this method private, i.e. start its name with a `_`